target/linux/ramips/image/lzma-loader/Makefile

```diff
@@ -11,13 +11,14 @@ include $(TOPDIR)/rules.mk
 LZMA_TEXT_START	:= 0x80a00000
 LOADER		:= loader.bin
 LOADER_NAME	:= $(basename $(notdir $(LOADER)))
-LOADER_DATA 	:=
```

Comment:
a small nit; (unrelated) commits are preferred in their own commits, so there's nice atomic commits :)